feature-toggle/src/app/demo/demo.component.ts

```diff
@@ -0,0 +1,41 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { DialogComponent } from '../dialog/dialog.component';
+
+import { MatDialog } from '@angular/material/dialog';
+
+
+
+interface Business {
```

Comment:
Move to seperate file, naming convention

Reply:
it will be removed in upcoming changes , it is used for handling mock data for testing